pkgs/applications/networking/plasma-dialer/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, mkDerivation
+, fetchFromGitLab
+
+, cmake
+, extra-cmake-modules
+
+, kcontacts
+, kcoreaddons
+, kdbusaddons
+, ki18n
+, kirigami2
+, knotifications
+, kpeople
+, libphonenumber
+, libpulseaudio
+, libqofono
+, protobuf
+, qtquickcontrols2
+, telepathy
+}:
+
+mkDerivation rec {
+  pname = "plasma-dialer";
+  version = "21.05";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "plasma-mobile";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "117hsd270ccpgvscx0s3p6a15islkqk6rh3b08ix931f4sapar2b";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  buildInputs = [
+    kcontacts
+    kcoreaddons
+    kdbusaddons
+    ki18n
+    kirigami2
+    knotifications
+    kpeople
+    libphonenumber
+    libpulseaudio
+    libqofono
+    protobuf # Needed by libphonenumber
```

Comment:
Should this be propagated instead?

Reply:
Good question, though note that all uses of `libphonenumber` add `protobuf` as an additional dependency.

I think either way it's a smell about the packaging of `libphonenumber`.